hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -1589,6 +1593,15 @@ public boolean checkAccess(OzoneObj ozObject, RequestContext context)
     Objects.requireNonNull(ozObject);
     Objects.requireNonNull(context);
     Objects.requireNonNull(context.getClientUgi());
+    String whiteList = conf.get(OZONE_WRITE_LIST);
+    Set<String> whiteListSet = new HashSet<>();
+    LOG.warn("=====keymanagerImpl=======checkAccess=====");
+    if (whiteList != null) {
+      whiteListSet.addAll(Arrays.asList(whiteList.trim().split(",")));
```

Comment:
This is going to rely on config to set up white list users, which ends up being static and requires reboot when changes to the lists happen. 
We want some CLI tool to dynamic modify the user list while the cluster is running and the list should be persisted somewhere so that it could load the list after reboot. So then concurrentHashSet may be a better option. Any updates to the list will have to go the concurrent hash set as well as to persistence. 
Also think about OM HA. Would this change be replicate to OM followers? Change to the white list is effectively a state change for OM.

Reply:
@timmylicheng Thanks for your review.
I think that's a good suggestion. Now the configuration of ozone. administrators is static, and om needs to be restarted for each update. I opened a task [HDDS-3591] to keep track of this.